### DISCUSSION TITLE
把TextReference翻译成文本引用

### DISCUSSION THREAD

**Discussion by dongrentianyu** [BramChen/tw5-docs#8] [General]
现在的版本是翻译成文本参考，但参考更有通过内容来指向其他内容的含义。我感觉翻译成文本引用会更好一点。因为引用就是引用了某一个条目的某个字段内容，可以是标签，正文，或者其他字段。

**Comment by BramChen** [comment]
翻譯之初，確實曾為兩者糾結過。
感謝先進指教。
